examples/mpi_plot_simulate_evoked.py

```diff
@@ -0,0 +1,41 @@
+"""
+===============
+Simulate dipole
+===============
+
+This example demonstrates how to run simulations with MPI using HNN-core.
```

Comment:
do you intend to add this example in the gallery? All examples that are part of the gallery should start with `plot_`. It's a sphinx_gallery thing

Reply:
Renamed in c4aec6134af2631969c187377e471a0ebe2a5008